website/content/api-docs/secret/databases/redis.mdx

```diff
@@ -0,0 +1,80 @@
+---
+layout: api
+page_title: Redis - Database - Secrets Engines - HTTP API
+description: >-
+  The Redis plugin for Vault's database secrets engine generates database
+  credentials to access Redis servers.
+---
```

Comment:
Should we say `... generates [ACL credentials](https://redis.io/docs/manual/security/acl/)...` to disambiguate with the default user credentials?

Reply:
Good point, it would be good to avoid the confusion. 👍